themes/_default.yaml

```diff
@@ -23,8 +23,11 @@ generic:
     accent-darker: &accent-darker   '#0B4848'
     accent-dark: &accent-dark       '#177D7D'
     accent: &accent                 '#2A9595'
+    accent-border: &accent-border   '#E6BB00'
```

Comment:
this is yellow, not Lochinvar color, wich we use for an accent color in the default theme